From bd33d38c2ed23465ba88fc5e56c97bb14df5ea9e Mon Sep 17 00:00:00 2001
From: Mark Grover <grover.markgrover@gmail.com>
Date: Wed, 19 Dec 2012 17:54:39 -0800
Subject: [PATCH 790/795] MR1: CLOUDERA-BUILD. CDH-8545: Package Job Tracker High Availability

(cherry picked from commit 8fca711ea5add34ca61c08579beb62959ab4504b)
---
 .../hadoop-common/src/main/bin/hadoop              |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/bin/hadoop b/hadoop-common-project/hadoop-common/src/main/bin/hadoop
index be91771..1a74cac 100755
--- a/hadoop-common-project/hadoop-common/src/main/bin/hadoop
+++ b/hadoop-common-project/hadoop-common/src/main/bin/hadoop
@@ -74,7 +74,7 @@ case $COMMAND in
     ;;
 
   #mapred commands for backwards compatibility
-  pipes|job|queue|mrgroups|mradmin|jobtracker|tasktracker)
+  pipes|job|queue|mrgroups|mradmin|jobtracker|tasktracker|mrhaadmin|mrzkfc|jobtrackerha)
     echo "DEPRECATED: Use of this script to execute mapred command is deprecated." 1>&2
     echo "Instead use the mapred command for it." 1>&2
     echo "" 1>&2
-- 
1.7.0.4

